### DISCUSSION TITLE
Only 1 Crazyflie is being detected by crazyswarm when using Lighthouse tracking system

### DISCUSSION THREAD

**Discussion by AvaneenP** [bitcraze/discussions#1247] [Q&A]
Hey everyone,

I'm using 4 crazyflies with the lighthouse tracking system. I've followed all the instructions detailed in the lighthouse start guide and have made the appropriate changes to the hover_swarm.launch file. However when I launch the aforementioned file, it seems like all the frames except for one crazyflie disappear in RVIZ and any further nodes I try to run to control the crazyflies do not pick up the other 3 CFs. Anyone face this problem and have found a solution?

**Comment by knmcguire** [comment]
Hi! If you are able to follow the[ lighthouse tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/) without a problem, then I think you'll better off asking this question in the [Crazyswarm2 discussions](https://github.com/IMRCLab/crazyswarm2/discussions) because I think this might be an issue for the CS server instead, which is outside of our ecosystem support.
